packages/backend/src/models/SavedChartModel.ts

```diff
@@ -1078,121 +1113,131 @@ export class SavedChartModel {
+                    "COALESCE(ARRAY_AGG(DISTINCT sqvtc.name) FILTER (WHERE sqvtc.name IS NOT NULL), '{}')",
+                ),
+                customMetrics: this.database.raw(
+                    "COALESCE(ARRAY_AGG(DISTINCT (sqvam.table || '_' || sqvam.name)) FILTER (WHERE sqvam.name IS NOT NULL), '{}')",
+                ),
+                customMetricsBaseDimensions: this.database.raw(
+                    "COALESCE(ARRAY_AGG(DISTINCT (sqvam.table || '_' || sqvam.base_dimension_name)) FILTER (WHERE sqvam.base_dimension_name IS NOT NULL), '{}')",
+                ),
+                customBinDimensions: this.database.raw(
+                    "COALESCE(ARRAY_AGG(DISTINCT sqvcd.id) FILTER (WHERE sqvcd.id IS NOT NULL), '{}')",
+                ),
+                customSqlDimensions: this.database.raw(
+                    "COALESCE(ARRAY_AGG(DISTINCT sqvcsd.id) FILTER (WHERE sqvcsd.id IS NOT NULL), '{}')",
+                ),
+                sorts: this.database.raw(
+                    "COALESCE(ARRAY_AGG(DISTINCT sqvs.field_name) FILTER (WHERE sqvs.field_name IS NOT NULL), '{}')",
+                ),
+            })
+            .from(cteName)
+            .leftJoin(
+                SavedChartVersionsTableName,
+                `${cteName}.saved_queries_version_id`,
+                'saved_queries_versions.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_fields as svf',
+                'saved_queries_versions.saved_queries_version_id',
+                'svf.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_table_calculations as sqvtc',
+                'saved_queries_versions.saved_queries_version_id',
+                'sqvtc.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_additional_metrics as sqvam',
+                'saved_queries_versions.saved_queries_version_id',
+                'sqvam.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_custom_dimensions as sqvcd',
+                'saved_queries_versions.saved_queries_version_id',
+                'sqvcd.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_custom_sql_dimensions as sqvcsd',
+                'saved_queries_versions.saved_queries_version_id',
+                'sqvcsd.saved_queries_version_id',
+            )
+            .leftJoin(
+                'saved_queries_version_sorts as sqvs',
+                'saved_queries_versions.saved_queries_version_id',
+                'sqvs.saved_queries_version_id',
+            )
+            .groupBy(1, 2, 3, 4, 5);
+
+        // Filter out charts that are saved in a dashboard and don't belong to any tile in their dashboard last version
+        const orphanedChartUuids = await this.getOrphanedChartUuids(
```

Comment:
We should rename the function. Orphan charts are charts that don't belong to any dashboard version while these,like your commend mentions, are charts not in the last dashboard version.
I have also renamed the PR title to not mention orphans.